amazon_transcribe/exceptions.py

```diff
@@ -61,12 +61,12 @@ def __init__(self, message):
         self.status_code: int = 503
 
 
-class ValidationException(SDKError):
-    """Encountered an issue validating a given value"""
+class SerializationException(ServiceException):
```

Comment:
So this is a bit odd. We're repurposing an SDKError to be a ServiceError because they share the same name. Is this being used anywhere else in the code already? If it is, this is technically breaking.

I think we can repurpose this since we're still expecting the core APIs to shift in alpha, but we need to a.) make sure this is set up correctly as a ServiceException with an `__init__` and b.) make sure we create a replacement SDKError for any uses this had from before.

Reply:
I think this exposing a more fundamental issue that we need to consider namespacing anything generated from the model into it's own module.

Reply:
That's potentially true. I think it makes sense from a generation standpoint, but is going to lead to a more painful user experience. If users want to catch these directly, they're going to have to understand the difference between importing service generated and SDK exceptions. Otherwise, we'll need some friendlier proxy for the two.